Source/ThirdParty/jsep.js

```diff
@@ -0,0 +1,599 @@
+//     JavaScript Expression Parser (JSEP) 0.3.0
```

Comment:
There's been some small changes since we pull this in - https://github.com/soney/jsep/commits/master

Could be worth doing a quick update so we ship with the latest.